tracing-opentelemetry/src/test/java/org/axonframework/tracing/opentelemetry/OpenTelemetrySpanTest.java

```diff
@@ -45,41 +46,45 @@ void setUp() {
             createdScopesList.add(mock);
             return mock;
         });
+        when(span.getSpanContext()).thenReturn(mock(SpanContext.class));
 
         openTelemetrySpan = new OpenTelemetrySpan(spanBuilder);
     }
 
     @Test
-    void startingTheSpanMakesItStartWithCurrentScope() {
+    void startingTheSpanStartsItButDoesNotMakeItCurrent() {
         openTelemetrySpan.start();
 
         verify(spanBuilder).startSpan();
-        verify(span).makeCurrent();
+        assertEquals(0, createdScopesList.size());
+    }
+
+    @Test
+    void startingTheSpanAndMakingItActiveMakesItCurrent() {
+        openTelemetrySpan.start();
+        openTelemetrySpan.makeCurrent();
+
+        verify(spanBuilder).startSpan();
```

Comment:
Why did you drop the `verify(span).makeCurrent();` validation here?

Reply:
Because starting a span will not automatically make it current. You need to open a scope manually, or use the supplied methods on the interface (runSupplier, run...)

Reply:
:thinking: so what's the `makeCurrent` invocation for on line 65 then? Doesn't that make the OpenTelemetry span current, as it did with the *original* test?

Reply:
I expanded the assertions, so that it is also checked whether the scope is closed. This should make the test more clear